common/src/main/java/com/wynntils/features/user/ItemScreenshotFeature.java

```diff
@@ -74,7 +73,7 @@ private static void takeScreenshot(Slot hoveredSlot) {
         List<Component> tooltip = stack.getTooltipLines(null, TooltipFlag.Default.NORMAL);
         WynnItemUtils.removeLoreTooltipLines(tooltip);
 
-        Font font = McUtils.mc().font;
+        Font font = FontRenderer.getInstance().getFont();
```

Comment:
What's the difference?

Reply:
Nothing at the moment, but I guess it is nice to keep this universal in case we change something up in the future.

Reply:
Maybe we should have a McUtils.getFont() instead?

Reply:
We could but I don't see any benefit atm. I imagine `McUtils.getFont()` could be the default Font for Minecraft, while our FontRenderer could have a Font with specific settings. We will see how this turns out in the future.

Reply:
Ok.